### PR TITLE
FEAT: Via fencing defeaturing

### PR DESCRIPTION
**Pull request by svandenb-dev**
New feature replacing via fence by single via.

### CODE REVIEW

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, left minor comments

File: src/pyedb/dotnet/edb_core/padstack.py [hunk 1]
```diff
@@ -1592,3 +1592,73 @@ def get_padstack_instances_intersecting_bounding_box(self, bounding_box, nets=No
...
+                    pad_shape="Polygon",
+                    antipad_shape="Polygon",
+                    pad_polygon=polygon_data,
+                    antipad_polygon=polygon_data,
+                    polygon_hole=polygon_data,
+                ):
+                    self._logger.error(f"Failed to create padstack definition {new_padstack_def}")
+                if not self.place(position=[0, 0], definition_name=new_padstack_def, net_name=net_name):
+                    self._logger.error(f"Failed to place padstack instance {new_padstack_def}")
+            [inst.delete() for inst in _instances_to_delete]
```
[the remark this answers is not in the document]
Comment (svandenb-dev) [reply src/pyedb/dotnet/edb_core/padstack.py:1665]: Yes ok

[the remark this answers is not in the document]
Comment (svandenb-dev) [reply src/pyedb/dotnet/edb_core/padstack.py:1665]: done

File: tests/legacy/system/test_edb_padstacks.py [hunk 2]
```diff
@@ -427,3 +427,13 @@ def test_polygon_based_padsatck(self):
         assert edbapp.padstacks.definitions["test"]
         assert edbapp.padstacks.definitions["test2"]
         edbapp.close()
+
+    def test_via_fence(self):
+        source_path = os.path.join(local_path, "example_models", test_subfolder, "via_fence_generic_project.aedb")
+        target_path = os.path.join(self.local_scratch.path, "test_pvia_fence", "via_fence.aedb")
+        self.local_scratch.copyfolder(source_path, target_path)
+        edbapp = Edb(target_path, edbversion=desktop_version)
+        assert edbapp.padstacks.merge_via_along_lines(net_name="GND", distance_threshold=2e-3, minimum_via_number=6)
```
[the remark this answers is not in the document]
Comment (svandenb-dev) [reply tests/legacy/system/test_edb_padstacks.py:436]: done

**Pull request merged**